devices/GpsDevice/category.txt

```diff
@@ -0,0 +1,2 @@
+gps
```

Comment:
not sure we have those categories yet, please check the project that builds automatically all the readmes! If needed, can you add those into the categoies? (sorry, a bit in a rush to check into details)